tools/harvester-forwarding/src/functions/getRecentDefinitions.js

```diff
@@ -0,0 +1,152 @@
+const { app } = require("@azure/functions");
+const { MongoClient } = require("mongodb");
+
+app.http("getRecentDefinitions", {
+  methods: ["GET"],
+  authLevel: "anonymous",
+  handler: async (request, context) => {
+    context.log(`Http function processed request for url "${request.url}"`);
+
+    try {
+      const daysBack = parseInt(request.query.get("days"));
+      if (!daysBack || isNaN(daysBack) || daysBack < 1 || daysBack > 30) {
+        return { status: 400, body: "days must be a number between 1 and 30" };
+      }
+      const limitPerTypeAndDay = parseInt(request.query.get("limit"));
+      if (
+        !limitPerTypeAndDay ||
+        isNaN(limitPerTypeAndDay) ||
+        limitPerTypeAndDay < 1 ||
+        limitPerTypeAndDay > 100
+      ) {
+        return {
+          status: 400,
+          body: "limitPerType must be a number between 1 and 100",
+        };
+      }
+
+      return await getData(context, daysBack, limitPerTypeAndDay);
+    } catch (error) {
+      context.log(`Error: ${error.message}`);
+      return { status: 500, body: `An error occurred: ${error.message}` };
+    }
+  },
+});
+
+async function getData(context, days, limitPerType) {
```

Comment:
Probably very unlikely it would happen.
If the Mongo DB schema was to change, I assume this test would fail and then whoever looked it would have to update.

But I doubt it will change this far into the project